_includes/comparison.html

```diff
@@ -107,7 +107,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
           <tr>
             <th scope="row">Drawbacks</th>
             <td>Istio's flexibility can be overwhelming for teams who don't have the capacity for more complex technology. Also, Istio takes control of the ingress controller.</td>
-            <td>Linkerd 2 is deeply integrated with Kubernetes and cannot be expanded. Since Linkerd 2 does not rely on a third-party proxy, it cannot be extended easily.</td>
+            <td>Linkerd 2 is deeply integrated with Kubernetes and does not currently support non-Kubernetes workloads.</td>
```

Comment:
It's true that Linkerd2-proxy doesn't currently support WASM, but that's intentional -- we haven't yet found a concrete use case for WASM yet, just a lot of marketing noise. So I wouldn't characterize it as a result of "not using a third-party proxy"—it's because we haven't built that feature. (Yet!)

But if that's the drawback you want to highlight, then how about I add: "It also does not currently support data plane extensions", since everything else is basically an implementation detail.

Reply:
I personally agree that currently, only very few teams benefit from WASM support and other envoy customization beyond the respective service mesh capabilities. But in the long run, it might be interesting for very specific use cases.

Your sentence is indeed much simpler and to-the-point than my proposal, please go ahead 👍